### ISSUE TITLE
Is it possible to sync the entire mongodb when my docker restarts?

### ISSUE THREAD

**Issue by JVictorV** [rwynn/monstache#320]
So I have three different dockers (api, monstache and mongodb). Monstache watches for new changes when it restarts, but if I already have any documents created before I start, I can't access those with elastic search, is there any way to index those old documents when i start the monstache docker?

**Comment by rwynn** [comment]
@JVictorV, yes it is.  See the docs for [direct-read-namespaces](https://rwynn.github.io/monstache-site/config/#direct-read-namespaces).  

```toml
direct-read-namespaces = [ "" ]
```
